dev/kind/start.sh

```diff
@@ -9,15 +9,13 @@ function cluster_exists {
 
 # Create the cluster.
 if ! cluster_exists; then
-  "${KIND}" create cluster --name "${CLUSTER_NAME}"
+  "${KIND}" create cluster \
+    --name "${CLUSTER_NAME}" \
+    --image "kindest/node:${KUBERNETES_VERSION}"
 else
   echo "Kind is already started"
 fi
 
-# Use the kubeconfig file generated by kind.
-KUBECONFIG="$("${KIND}" get kubeconfig-path --name "${CLUSTER_NAME}")"
-export KUBECONFIG
```

Comment:
We no longer to use this kubeconfig? Why?

Reply:
kind is doing like minikube.

Reply:
I take that to mean "it modifies your default kubeconfig to have a new context &c"? That's fine, then.

Reply:
Yep